starter-code/index.html

```diff
@@ -2,30 +2,49 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-  Instagram
+  <div class="phone">
+    <img src="images/phones.png" alt="phones">
```

Comment:
To all of your relative links on your index.html, change `"images/phones.png"` to adding a `./` before, so like this: `./"images/phones.png"`